websites/F/Formuliser/dist/metadata.json

```diff
@@ -0,0 +1,20 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.0",
+  "author": {
+    "name": "D̲u̲c̲k̲o̲ 🌈",
+    "id": "506482395269169153"
+  },
+  "url": "formuliser.oojmed.com",
+  "description": {
+    "en": "Get information on chemical formulae."
+  },
+  "service": "Formuliser",
+  "version": "1.0.0",
+  "logo": "https://i.imgur.com/rtkOdlB.png",
+  "thumbnail": "http://i.imgur.com/nEJz6j6.png",
```

Comment:
This needs to be HTTPS.

Reply:
Weird it isn't, fixing it in new commit now.